docs/articles/blog/2017-6-7-testcafe-v0-16-0-released.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: TestCafe v0.16.0 Released
+permalink: /blog/:title.html
+---
+# TestCafe v0.16.0 Released
+
+TypeScript support, seamless testing in headless Chrome and device emulator, and numerous bug fixes.
+
+<!--more-->
+
+## Enhancements
+
+### ⚙ TypeScript support ([#408](https://github.com/DevExpress/testcafe/issues/408))
+
+In this release, we have added an ability to write tests in [TypeScript](https://www.typescriptlang.org/). Using of TypeScript brings you all the advantages of strongly typed languages: rich coding assistance, painless scalability, check-as-you-type code verification and much more.
+
+TestCafe bundles TypeScript declaration file with the npm package, so you have no need to install any additional packages.
+
+Just create a `.ts` file with the
+
+<!-- Use `js` instead of `ts` for this code block for proper code highlighting -->
+
+```js
+import { Selector } from 'testcafe';
+```
+
+and write your test.
+
+For details, see [TypeScript Support](https://devexpress.github.io/testcafe/documentation/test-api/typescript-support.html)
+
+### ⚙ Support running in Chrome in headless mode and in device emulator ([#1417](https://github.com/DevExpress/testcafe/issues/1417))
+
+Now TestCafe allows you to run your tests in Google Chrome in the headless mode and the device emulation mode.
+
+[Headless mode](https://developers.google.com/web/updates/2017/04/headless-chrome) allows you to run tests in Chrome without any visible UI shell. To run tests in the headless mode use the `:headless` postfix:
+
+```sh
+testcafe "chrome:headless" tests/sample-fixture.js
+```
+
+Device emulation mode allows you to check how your tests works on mobile devices via Chrome's built-in [device emulator](https://developers.google.com/web/tools/chrome-devtools/device-mode/). To run tests in the device emulation mode specify `emulation:` and [device parameters](https://devexpress.github.io/testcafe/documentation/common-concepts/browser-support.md#available-chrome-options):
```

Comment:
... To run tests in device emulation mode, specify `emulation:` and [device parameters](https://devexpress.github.io/testcafe/documentation/common-concepts/browser-support.md#available-chrome-options):